src/services/__tests__/cartService.test.ts

```diff
@@ -0,0 +1,68 @@
+import { Product } from '../../domain/Product';
```

Comment:
Is there some rule of the naming convention of files so we have import **P**roduct  from **P**roduct and **I**nMemoryCartService from **i**nMemoryCartService (lowercased first char)?

Reply:
We have not developed any so far.